.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "wasmCloud",
+    "image": "rust:bullseye",
+    "features": {
+        "devwasm.azurecr.io/dev-wasm/dev-wasm-feature/rust-wasi:0": {},
+        "ghcr.io/lee-orr/rusty-dev-containers/cargo-binstall:latest": {},
+        "ghcr.io/lee-orr/rusty-dev-containers/cargo-watch:latest": {},
+        "ghcr.io/lee-orr/rusty-dev-containers/cargo-nextest:latest": {},
```

Comment:
Like mentioned in https://github.com/wasmCloud/wasmCloud/pull/335 , these seem like non-essential functionality, so I'm curious why it's here?

Reply:
I'll replicate your comments & my responses from the original PR here - note that it seems that github considered my responses as though they were an un-submitted review, so they were only visible to me...